src/show.jl

```diff
@@ -22,14 +22,31 @@ Base.show(io::IO, p::TypesWithShow) = show(io, MIME"text/plain"(), p)
 # VARIABLES
 function _show(io::IO, mime::MIME, var::AbstractVariable)
     base, indices = name_base_indices(var)
-    if isempty(indices)
-        print(io, base)
+    if isconj(var)
+        for c in base
+            print(io, c, '\u0305')
+        end
     else
         print(io, base)
+    end
+    if !isempty(indices)
         print_subscript(io, mime, indices)
     end
+    isrealpart(var) && print(io, "ᵣ")
+    isimagpart(var) && print(io, "ᵢ")
+end
+
+function _show(io::IO, mime::MIME"text/print", var::AbstractVariable)
+    if isconj(var)
+        for c in name(var)
+            print(io, c, '\u0305')
```

Comment:
Could you add comment for what this is doing ?

Reply:
Say we create a complex-valued variable `z`. Then, its conjugate is displayed as bar-z (z with overbar).

Reply:
Nice, adding that as comment in the code could help